packages/push/README.md

```diff
@@ -0,0 +1,5 @@
+## AeroGear Services Unified Push Registration SDK
```

Comment:
I think we should remove _Registration_ because it's not _only_  about _Registration_. Wdyt?

Reply:
It is only about registration, all other functionalities will be handled in the plugin. The whole purpose of our SDK should be to enable/simplyfy UPS registration